aiidalab/app.py

```diff
@@ -572,9 +578,9 @@ def install(
             except RuntimeError:
                 logger.exception("Rollback failed! Consider re-installing the app.")
             finally:
-                raise RuntimeError(
-                    f"Failed to install '{self.name}' (version={version}) at '{self.path}'"
-                ) from error
+                logger.info("Full stack trace", exc_info=True)
```

Comment:
This displays the full stack trace when a larger verbosity is used (`aiidalab -v install`)